requirements.txt

```diff
@@ -12,3 +12,4 @@ PyYAML==5.3.1
 ansicolors==1.1.8
 pyusb==1.1.0
 six>=1.13.0
+python-vxi11==0.9
```

Comment:
Hmm, this way, it will be installed for every labgrid user, although probably only a few will use it.

Maybe wrap the import in a `try:`/`except:` and tell the user to install the module? You could document the requirement in the NetworkPowerDriver section in doc/configuration.rst, as well.

Reply:
True.

But it is a pure-python module, 2144 lines of code (including comments and blank lines), and no dependencies.
Are you sure it is worth making it more difficult to use to avoid pulling it in?

It is worth noticing that the protocol is not Siglent specific, but could be used to extend support to many other VXI-11 compatible instruments.

If you still think that I should wrap the import, where do you want that done?  In siglent.py or i powerdriver.py?

Reply:
In siglent.py. For special cases like this, we also have {modbus,onewire,snmp,xena}-requirements.txt. So maybe a vxi-requirements? And also add it to dev-requirements so that you get all of them for development.

Reply:
Ok.

But shouldn't it suffice to move the dependency to vxi11-requirements.txt?
The siglent.py module is only going to be imported when a `NetworkPowerPort` is configured with `modul: siglent`.
And cannot come up with something much more sensible than just raising an exception, which the import does anyway.

If you take the `onewire` module example, it simply calls
`
    self._module = import_module('onewire')
`

which end up throwing an exception, just as the import in siglent.py will.

Reply:
You're right. That's fine.